tests/nest_tests/stdp_triplet_synapse_test.py

```diff
@@ -470,32 +456,19 @@ def _test_stdp_triplet_synapse(delay, spike_times_len):
     compare_results(ref_timevec, ref_w, nestml_timevec, nestml_w)
 
 
-# @pytest.mark.parametrize("spike_times_len", [1, 10, 100])
 @pytest.mark.parametrize("spike_times_len", [10])
 def test_stdp_triplet_synapse_delay_1(spike_times_len):
     delay = 1.
     _test_stdp_triplet_synapse(delay, spike_times_len)
 
 
-logging.warning("XXX: TODO: xfail test due to https://github.com/nest/nestml/issues/703")
-# @pytest.mark.xfail(strict=True, raises=Exception)
-# @pytest.mark.parametrize("spike_times_len", [1, 10, 100])
-
-
 @pytest.mark.parametrize("spike_times_len", [10])
 def test_stdp_triplet_synapse_delay_5(spike_times_len):
-    logging.warning("XXX: TODO: this test is disabled due to https://github.com/nest/nestml/issues/703")
-    # delay = 5.
-    # _test_stdp_triplet_synapse(delay, spike_times_len)
-
-
-logging.warning("XXX: TODO: xfail test due to https://github.com/nest/nestml/issues/703")
-# @pytest.mark.xfail(strict=True, raises=Exception)
-# @pytest.mark.parametrize("spike_times_len", [1, 10, 100])
+    delay = 5.
```

Comment:
Could you parametrize the `delay` as well?

Reply:
Done!